src/methods/index.ts

```diff
@@ -1,9 +1,9 @@
 import * as balances from './balances';
 import * as claims from './claims';
-import * as democracy from './democracy';
```

Comment:
I would say to keep all calls inside `democracy`, some people might still be using it. A console.warn inside these functions should be enough.

Reply:
Yeah, these are still active on Kusama so we need them (although a runtime upgrade is in that will replace them). Also we don't want to remove `vote` from democracy, just the proxy stuff.

Reply:
Actually, I'm too slow for Kusama. That runtime upgrade already went through. So for Kusama/Polkadot, we just need `democracy.vote`, but agree we should leave the others in for other chains and mark them as deprecated.

Reply:
I put back in all democracy methods, but had to put in deprecated metadata in order to keep the tests passing.

Reply:
Let's add a console.warn in the deprecated functions too. Even if other chains are still using them, it'll be a gentle reminder for them to update to latest substrate

Reply:
> but had to put in deprecated metadata in order to keep the tests passing

In order to keep this repo easier to maintain, I would also be fine with removing tests for deprecated methods. @emostov I'll let you decide on this one.